test/e2e/tests/settings/full-size-view-settings.spec.js

```diff
@@ -33,28 +33,25 @@ describe('Full-size View Setting @no-mmi', function () {
         await unlockWallet(driver);
         await toggleFullSizeViewSetting(driver);
         await openDapp(driver);
```

Comment:
This file assumed that window handles at certain indices would always map to the dapp and popup window, but again the offscreen document can be at one of these indices.

The solution, which maintains what this test was actually testing, is to update it to compare window handles before and after the user action being tested, and to make an assertion against the new window handle